.gitignore

```diff
@@ -12,6 +12,9 @@
 # Built Visual Studio Code Extensions
 *.vsix
 
+# Binary:
+bin
```

Comment:
is the binary file itself called bin? Shouldnt we instead give in an extension such as "output.bin" and gitignore *.bin files?

Reply:
I had a local directory called bin that has the ``act`` program in it for running our workflows without commits/pushing/PRs so I added it to the .gitignore to not clutter the remote. How do you want me to proceed?

Reply:
if its a directory, not a file, thats perfect, resolving this comment.